wagtailmenus/tests/test_disabling_menus_in_cms_settings.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
For organisation's sake, I also feel that `test_backend.py` would be a better home for these tests (even if they can't be simplified)